src/index.ts

```diff
@@ -1,42 +1,56 @@
 import Client from './client';
+import LRU from 'lru-cache';
 import { Options, Target } from './types';
 import { Logger } from './log';
-import { KeyValueStore } from './types';
+import { AsyncKeyValueStore, KeyValueStore } from './types';
+import { FileStore } from './store';
 
-export { Client, Options, Target, KeyValueStore, Logger };
+export {
+  Client,
+  Options,
+  Target,
+  AsyncKeyValueStore,
+  KeyValueStore,
+  Logger,
+  LRU,
+  FileStore,
+};
 export default {
   instance: undefined,
   init: function (sdkKey: string, options: Options): void {
     if (!this.instance) {
       this.instance = new Client(sdkKey, options);
     }
   },
-  boolVariation: function (
+  boolVariation: async function (
```

Comment:
Since you're not using `await`, you don't need `async`, unless you want to add `await` to the `this.instance.boolVariation` call